MultiTable/src/main.js

```diff
@@ -45,6 +45,13 @@ let subjectModal;
 let aspectModal;
 let urlParams = {};
 
+// for (ctrl + f) or (cmd + f)
```

Comment:
The PR should also include the update to the version and release notes in the lens.json, plus build the lens to generate the zip file, plus include the new zip file in the PR